writeengine/wrapper/writeengine.cpp

```diff
@@ -1026,6 +1258,12 @@ int WriteEngineWrapper::insertColumnRecs(const TxnID& txnid,
     for (i = 0; i < colStructList.size(); i++)
         Convertor::convertColType(&colStructList[i]);
 
+    for (i = 0; i < colStructList.size(); i++)
```

Comment:
I believe there is [vector ctor](https://en.cppreference.com/w/cpp/container/vector/vector) that allows to fill with a class instances when called.

Reply:
The habit of copying general approach of code around sometimes fails me.

The code needs a mapper (which I hadn't added yet) for new structure constructor. So, it is an iterator loop now.